bitcoin/src/util/psbt/mod.rs

```diff
@@ -437,6 +437,29 @@ impl PartiallySignedTransaction {
         // because there has been a new softfork that we do not yet support.
         Err(SignError::UnknownOutputType)
     }
+
+    /// Calculates fee
+    ///
+    /// ## Errors
+    ///
+    /// The function returns [`Error::MissingUtxo`] when UTXO information for at least one input
+    /// is not present or is invalid.
+    /// If calculated value is negative [`Error::NegativeFee`] is returned.
+    pub fn fee(&self) -> Result<u64, Error> {
+        let mut inputs = 0;
+        for utxo in  self.iter_funding_utxos() {
+            inputs += match utxo {
```

Comment:
Overflow is not handled. I suggest explicitly checking and returning `Overflow` error.

Reply:
fixed, added an error variant